### PR TITLE
Update ja/docs/tutorials/stateless-application/expose-external-ip-add…

### LINKED ISSUE #21367: ja: Make tutorials/stateless-application/expose-external-ip-address.md follow v1.17 of the original text
**This is a Feature Request**

**What would you like to be added**
Update tutorials/stateless-application/expose-external-ip-address.md to follow v1.17 of the original (Englist) text.

**Why is this needed**
content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md is outdated.

**Comments**
/language ja
/good-first-issue

File to update
https://github.com/kubernetes/website/blob/dev-1.17-ja.2/content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md

https://kubernetes.io/ja/docs/tutorials/stateless-application/expose-external-ip-address/

Original
https://github.com/kubernetes/website/blob/release-1.17/content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

https://v1-17.docs.kubernetes.io/docs/tutorials/stateless-application/expose-external-ip-address/

diff between translated and v1.17
https://gist.github.com/oke-py/d6592eae1a742bc61623eb94f0adc117

**Note**
Currently, we use `dev-1.17-ja.2` branch. Please open a PR targeting the branch.

ref
[How To Contribute](https://kubernetes-docs-ja.kibe.la/shared/entries/c5878aa5-ad1f-4f29-a5bb-25853cbc14ec)
[翻訳スタイルガイド](https://kubernetes-docs-ja.kibe.la/shared/entries/5efe4fa7-d2a1-4a1d-8bc3-ce7ccdc064a6)

If you have a question, feel free to ask us at slack.k8s.io #kubernetes-docs-ja channel.

### PR DESCRIPTION
**Pull request by ryok-0319**
close #21367

### CODE REVIEW

**Comment by ryok-0319** [comment]
I signed it

**Comment by ryok-0319** [comment]
I signed it

**Comment by ryok-0319** [comment]
I signed it

File: content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md [hunk 1]
```diff
@@ -69,7 +69,17 @@ kubectl apply -f https://k8s.io/examples/service/load-balancer-example.yaml
         NAME         TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)    AGE
         my-service   LoadBalancer   10.3.245.137   104.198.205.71   8080/TCP   54s
 
-    注意: 外部IPアドレスが\<pending\>と表示されている場合は、しばらく待ってから同じコマンドを実行してください。
+    {{< note >}}
+
+    `type=LoadBalancer`のServiceは外部のクラウドプロバイダーによってサポートされており、この例ではカバーされていません。詳細は[このページ](/ja/docs/concepts/services-networking/service/#loadbalancer)を参照してください。
```
Comment (inductor) [inline content/ja/docs/tutorials/stateless-application/expose-external-ip-address.md:74]:
```suggestion
    `type=LoadBalancer`のServiceは外部のクラウドプロバイダーによってサポートされており、ここでは扱いません。詳細は[こちらのページ](/ja/docs/concepts/services-networking/service/#loadbalancer)を参照してください。
```

**Pull request merged**